packages/portfolio-deploy/src/portfolio-start.core.js

```diff
@@ -1,7 +1,7 @@
 // import { meta } from '@aglocal/portfolio-contract/src/portfolio.contract.meta.js';
 import { makeTracer } from '@agoric/internal';
 import { M } from '@endo/patterns';
-import { localchainContracts } from './axelar-configs.js';
+import { axelarConfigMock } from './axelar-configs.js';
```

Comment:
We shouldn't be using a Mock except in a test.

The config info should be passed in, not imported.